.github/workflows/update-kubernetes.yaml

```
@@ -31,9 +31,15 @@ jobs:
         labels: |
           automated-pr
           kubernetes
+          type::chore
         draft: false
         base: "main"
-        body: "Automated changes by the [cron-kubernetes-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-kubernetes.yaml) GitHub action"
+        body: |
+          Automated changes by the [cron-kubernetes-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-kubernetes.yaml) GitHub action
+          
+          ```release-note
+            Adds [Kubernetes](https://kurl.sh/docs/add-ons/kubernetes) version ${{ steps.update.outputs.kubernetes_version }}
```

```suggestion
            Adds [Kubernetes](https://kurl.sh/docs/add-ons/kubernetes) version(s) ${{ steps.update.outputs.kubernetes_version }}
```

https://github.com/replicatedhq/kURL/pull/3303